articles/personal_growth.md

```diff
@@ -0,0 +1,33 @@
+Very much the same also applies to us, people who are involved in project work, where we try to give our best effort to make the project a success. 
+
+We work in projects as a subject matter expert, as an experienced project manager or as a newcomer that will need support from the team to grow to a new leader. Roles and tasks are assigned based on experiences we made in previous assignments. Experiences from former projects are our soil. We are able to grow on new and demanding tasks but will fail on overwhelming tasks. 
+If our responsibility lies within our comfort zone and is business as usual, we will not reach our limits. We need challenges such as finding new solutions and new approaches to grow. Just as a plant that needs more and more space to become bigger and greater we need space and the opportunity to make our own decisions and take responsibility.
+We need managers who see the potential of the project but also required skills for staffing such a project with team members. Managers that are aware of the strengths and weaknesses of each person and if there is a realistic feasibility that person can perform well and do a great job within the project.
+
+Besides the soil a plant needs right at the beginning also some kind of pillar. As long as the roots are not deeply anchored, and the stem is still fragile it is important that the plant is supported. It is exactly like that with us trying to grow in our projects. By having all these challenges and opportunities to grow, sometimes we will make a mistake. Sometimes we are not fulfilling expectation of our stakeholders, maybe our estimation or planning was wrong due to wrong assumptions, we took wrong decisions or communicated not well or the way we acted was not as good as expected from our sponsor. These are the moments we need support and orientation when this situation takes us to our limits. We need someone taking care of us, helping that we will not break in the storm but become stronger and are prepared for the next time we face a similar situation.
+
+Most often we expect the project manager to support and stand up for us, but this is not always the case. This may be because of personal reasons or up to political topics like internal - external employee. From my perspective that doesn't matter if it is the PL or anyone else, as long as I know right from the beginning if I can count on this person or who is going to support me. This supporting person should be within the project stakeholders or at least within the area affected by my activities. It could be the PL, the sponsor, a team lead, someone from operation, business representatives and so on. Crucial is, that this person is aware of the project and what's going on, that I am linked and in close contact with the person, that I have an open and transparent communication to the person and that the person is a supporter of mine. For this reason, you should setup bilateral meetings or ask for feedback .
```

Comment:
"From my perspective that doesn't matter if it is the PL or anyone else, as long as I know right from the beginning if I can count on this person or who is going to support me." -> "From my perspective it doesn't matter **which role supports me**, as long as I know right from the beginning **that** I can count on **a** person who is going to support me."

Grammatik (Space): "feedback ." -> "feedback."